### PR TITLE
Feat/sc 137 data for screen readers

### PR DESCRIPTION
**Pull request by davidgilbertson**
Jira: https://govausites.atlassian.net/browse/SC-137

For screen readers, we will replace this:
image

with this:
image

### CODE REVIEW

File: src/components/CityColumnChart/ChartTable/ChartTable.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+const ChartTable = props => (
+  <table className={props.className}>
+    <caption>
+      Table showing {props.chart.name} for each city. {props.chart.description}
+    </caption>
+
+    <thead>
+      <tr>
+        <th scope="col">City</th>
```
Comment (davidgilbertson) [inline src/components/CityColumnChart/ChartTable/ChartTable.js:14]: The `scope` attribute added as suggested here: https://webaim.org/techniques/tables/data

File: src/components/CityColumnChart/CityColumnChart.js [hunk 2]
```diff
@@ -310,62 +311,71 @@ class CityColumnChart extends Component {
...
+          </div>
 
-        <div className={style.chartWrapper}>
-          <div id={this.chartDivId} />
+          <div className={style.chartWrapper}>
+            <div id={this.chartDivId} />
+          </div>
         </div>
-      </div>
+      </article>
```
Comment (davidgilbertson) [inline src/components/CityColumnChart/CityColumnChart.js:378]: This looks like a lot changes but it's just indenting and a few new semantic tags, and the new `<ChartTable>`

**Comment by jonalport** [comment]
LGTM

File: src/components/Header/Header.scss [hunk 3]
```diff
@@ -1,13 +1,8 @@
 @import '~style/variables';
 
-header {
+.header {
```
Comment (davidgilbertson) [inline src/components/Header/Header.scss:3]: Some erroneous styles directly on tags broke when I added more headers to the site.

File: src/constants/indicators.js [hunk 4]
```diff
@@ -246,15 +246,13 @@ export default {
...
     name: 'Indigenous unemployment rate',
-    legendText: 'Indigenous',
     format: FORMATS.PERCENT_ZERO_TO_TWO_DECIMALS,
     contextual: false,
     lastUpdated: '2016',
     source: 'Unemployment rate, Unemployed, % of indigenous population',
   },
   unemploymentRateYouth: {
     name: 'Youth unemployment rate',
-    legendText: 'Youth',
```
Comment (davidgilbertson) [inline src/constants/indicators.js:257]: There's had mistakenly been left in the data from a previous change.

File: src/style/_misc.scss [hunk 5]
```diff
@@ -19,3 +19,14 @@ $z-index-tooltip: 5;
     box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
   }
 }
+
+@mixin hideText {
```
Comment (jonalport) [inline src/style/_misc.scss:23]: Was thinking this should maybe be something more generic, as you could be hiding something other than text?

I had it as `@mixin hidden`. No biggie.

**Comment by jonalport** [comment]
I will pull this down and go over it in some more detail.

**Comment by jonalport** [comment]
I'm able to navigate the tables well using VO, but I'm getting this weird thing where columns are read twice. Eg)

image

image

image

image

Have you noticed that? Seems like it would be better if they were just read once...

**Review by jonalport — APPROVED** [review]
Hiya. Looks good apart from that duplication thing. I will now submit my review...

File: src/components/Checkbox/Checkbox.scss [hunk 6]
```diff
@@ -48,6 +48,7 @@ $box-size: 16px;
   height: 12px;
   border: 2px solid $color-white;
   transform: rotate(45deg);
+  backface-visibility: hidden; // required to handle a bug in Chrome
```
Comment (jonalport) [inline src/components/Checkbox/Checkbox.scss]: Well there's something I've never seen before!

Comment (davidgilbertson) [reply src/components/Checkbox/Checkbox.scss]: I've reverted this - I just committed it to the wrong branch by mistake. After spending the weekend writing a blog post about not screwing up with git...

Comment (jonalport) [reply src/components/Checkbox/Checkbox.scss]: Lol yes, I saw that blog post, it was good. I need to chat with you about Dennis Chopper.

File: src/style/_misc.scss (hunk 5 above)
Comment (davidgilbertson) [reply src/style/_misc.scss:23]: Good idea, I've updated it to match yours.

**Pull request merged**